### PR TITLE
Disable MSRP option

### PR DESCRIPTION
**Pull request by cdosoftei**
MSRP is enabled by default; this PR implements a configuration option so the default behaviour can be changed when desired.

Also addresses signalwire/freeswitch#22

### CODE REVIEW

**Comment by mjerris** [comment]
I think i'd rather have this where if you don't configure the msock ports we don't listen on them, instead of a different setting.

**Comment by cdosoftei** [comment]
:+1: That would make it consistent with other configuration options (ws and wss ports for Sofia profiles come to mind).

Can definitely make the change -- @seven1240 do you wish to weigh in (given the feedback in #22)?

**Comment by seven1240** [comment]
@cdosoftei I totally agree with @mjerris, the only concern is that will change behaviour for existing users using msrp. we should alert behaviour change on the next release.

**Comment by cdosoftei** [comment]
Please check the [revised PR](https://github.com/signalwire/freeswitch/pull/36/files), it should be in line with the most recent feedback.

To @seven1240's point, the configuration behaviour change should be documented in the release notes (and perhaps in Confluence as well?); the vanilla configuration is not disrupted by this PR as it would yield the same behaviour, i.e. both the plain text and the secured listeners would be enabled.

File: conf/vanilla/autoload_configs/msrp.conf.xml [hunk 1]
```diff
@@ -3,7 +3,6 @@
     <param name="listen-ip" value="$${local_ip_v4}"/>
     <param name="listen-port" value="2855"/>
     <param name="listen-ssl-port" value="2856"/>
-    <!-- <param name="disable-msrp" value="true"/> -->
```
Comment (andywolk) [inline conf/vanilla/autoload_configs/msrp.conf.xml]: That was added by previous commit and now removed? How so?

**Pull request merged**